agent/src/main/java/com/code_intelligence/jazzer/agent/Agent.kt

```diff
@@ -103,7 +104,9 @@ fun premain(agentArgs: String?, instrumentation: Instrumentation) {
         ManifestUtils.combineManifestValues(ManifestUtils.HOOK_CLASSES).flatMap {
             it.split(':')
         }.filter { it.isNotBlank() }
-    val customHookNames = manifestCustomHookNames + (argumentMap["custom_hooks"] ?: emptyList())
+    val customHookNames = (manifestCustomHookNames + (argumentMap["custom_hooks"] ?: emptyList())).filter {
+        it !in (argumentMap["disabled_hooks"]?.toSet() ?: emptySet())
```

Comment:
An info log could be nice to detect incidentally disabled hooks.

Reply:
Good point, added.